### PR TITLE
Add server type

### PR DESCRIPTION
**Pull request by orblazer**
Hello,
This add a server type like your other project https://github.com/itzg/docker-minecraft-server

This fix issue #14.

Thanks.

### CODE REVIEW

File: run-bungeecord.sh [hunk 1]
```diff
@@ -1,13 +1,55 @@
...
+    : ${BUNGEE_JAR_REVISION:=${BUNGEE_JOB_ID}}
+    BUNGEE_JAR=$BUNGEE_HOME/${BUNGEE_JAR!=BungeeCord-${BUNGEE_JAR_REVISION}.jar}
+  ;;
+
+  WATERFALL)
+    : ${BUNGEE_BASE_URL:=https://papermc.io/ci/job/Waterfall/}
+    : ${BUNGEE_JOB_ID:=lastStableBuild}
+    : ${BUNGEE_JAR_URL:=${BUNGEE_BASE_URL}/${BUNGEE_JOB_ID}/artifact/Waterfall-Proxy/bootstrap/target/Waterfall.jar}
+    : ${BUNGEE_JAR_REVISION:=${BUNGEE_JOB_ID}}
+    BUNGEE_JAR=$BUNGEE_HOME/${BUNGEE_JAR!=Waterfall-${BUNGEE_JAR_REVISION}.jar}
```
Comment (itzg) [inline run-bungeecord.sh:32]: Looks like this should be
```suggestion
    BUNGEE_JAR=$BUNGEE_HOME/${BUNGEE_JAR:=Waterfall-${BUNGEE_JAR_REVISION}.jar}
```

Comment (orblazer) [reply run-bungeecord.sh:32]: Oh yes that is a misstake

**Comment by itzg** [comment]
PS: I recommend avoiding force pushing to PRs since it interferes with the review process.

**Comment by orblazer** [comment]
OK its noted

**Pull request merged**